openwisp-qa-check

```diff
@@ -101,7 +101,7 @@ runblack() {
 
 runcheckcommit() {
   if [ -z "$COMMIT_MESSAGE" ]; then COMMIT_MESSAGE=$(git log -1 --pretty=%B); fi
-  if [ "$TRAVIS" = true ] && [ "$TRAVIS_PULL_REQUEST" = false ]; then
+  if [[ "$COMMIT_MESSAGE" =~ ^Bump ]] || ([ "$TRAVIS" = true ] && [ "$TRAVIS_PULL_REQUEST" = false ]); then
```

Comment:
look here: https://github.com/openwisp/openwisp-utils/blob/master/openwisp_utils/qa.py#L98
That's where we should do it, please also add a test for it.

Reply:
Thanks for reviewing, @nemesisdesign please take a look now.